### DISCUSSION TITLE
Add Option/Operating Mode for Backwards-Facing Cars and Trains

### DISCUSSION THREAD

**Discussion by kyphii** [OpenRCT2/OpenRCT2#19306] [Feature requests] [answered]
Feature Concept: Implement new options or operating mode to allow coaster trains and/or cars to be run backwards. This feature would be simpler and more flexible than current hack-based methods for backwards builds. It could also superannuate the ride vehicle-based implementation currently used by ride types that support this, so it could eliminate some redundant vehicle sprites. This could also allow all ride vehicles to properly use the turntable/heartline transfer/reverser pieces.

I made a proof of concept for it over the past couple days just to see if it was possible, and it seems to work fine (though it does touch a fair bit of vehicle drawing code). Talked about it on the discord with some folks and feedback seemed mostly positive, though there's some room for different design options. Interested to get some feedback here.

**Comment by jensj12** [comment]
Duplicate of #15701

**Reply by J-PIE-314** [reply, marked as answer]
And implemented in PR https://github.com/OpenRCT2/OpenRCT2/pull/19305.
